src/main/java/com/yahoo/memory/WritableBuffer.java

```diff
@@ -348,4 +355,13 @@ public abstract void putBooleanArray(boolean[] srcArray, int srcOffsetBooleans,
    */
   public abstract long getRegionOffset();
```

Comment:
Why this method needs to be a part of public API? It's spec is complex and exposes too much internal details.

Reply:
There are legitimate cases where we need to extract the region offset and the underlying array, which avoids an array copy.  This call is only available from Writable *.